testing/todocheck_test.go

```diff
@@ -330,6 +331,50 @@ func TestInvalidIssueTracker(t *testing.T) {
 	}
 }
 
+func TestInvalidOrigins(t *testing.T) {
+	invalidConfigPaths := []string{
+		"./test_configs/invalid_jira_origin.yaml",
+		"./test_configs/invalid_github_origin.yaml",
+		"./test_configs/invalid_gitlab_origin.yaml",
+		"./test_configs/invalid_pivotal_origin.yaml",
+		"./test_configs/invalid_redmine_origin.yaml",
+	}
+
+	for _, path := range invalidConfigPaths {
+		cfg, err := config.NewLocal(path, ".")
+		if err != nil {
+			t.Errorf("%s", err)
+			continue
+		}
+		errors := cfg.Validate()
+		if 0 == len(errors) {
+			t.Errorf("%s should be invalid", path)
+		}
+	}
+}
+
+func TestValidOrigins(t *testing.T) {
+	validConfigPaths := []string{
+		"./test_configs/valid_jira_origin.yaml",
+		"./test_configs/valid_github_origin.yaml",
+		"./test_configs/valid_gitlab_origin.yaml",
+		"./test_configs/valid_pivotal_origin.yaml",
+		"./test_configs/valid_redmine_origin.yaml",
```

Comment:
This is a pretty neat way to test all these.
It avoids adding a separate test case for all issue trackers.

Thanks!